### ISSUE TITLE
Templates. Form elements must have labels

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#60]
## Description
Bulk template selection doesn't provide accessible text to checkboxes

![image](https://user-images.githubusercontent.com/5230720/135674829-423274c6-3fbf-444d-958c-a1c2c13917a6.png)

## Possible fix
We have to add accessible text to checkboxes, even if the label is not visible to users. 

## Resources
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Comment by amazingphilippe** [comment]
@YedidaZalik Wondering whether the accessible name for the checkbox should be simply:  

"Select [item name]"

@EliseKa Et en français: "Sélectionner [nom de l'item]"

**Comment by amazingphilippe** [comment]
These checkboxes act different from clicking on the name of the item. They're used to bulk select items

**Comment by EliseKa** [comment]
Good idea to use the item name as a variable! Is "folder" already specified in the instruction? If not it might be a good idea to be more specific and say "Select the [item name] folder" / "Sélectionner le dossier [item name]"

**Comment by amazingphilippe** [comment]
There might be a way! Some items are folders, others are templates. Depending if I can tell them apart, I may be able to set "Select the [item name] folder" for folders and "Select the [item name] template" for templates
